src/Runtime/python/onnxmlirtorch/src/onnxmlirtorch/onnxmlirtorch.py

```diff
@@ -0,0 +1,199 @@
+import numpy as np
+import os
+import sys
+import tempfile
+import torch
+
+import onnxmlir
+from .sessioncache import SessionCache
+
+"""
+This file provides the utility to run inference of a torch model with onnx-mlir
+compiler. There are two ways:
+
+1. Add a wrapper to a torch model so that the onnx export, compile with 
+onnx-mlir and run will happen automatically at inference of the model
+
+Example code:
+ 
+ # Assuem torch_model is the a torch model
+ model = ONNXLIRTorch(torch_model)
+ results = torch_model(inputs) 
```

Comment:
```suggestion
 results = model(inputs) 
```
Did you mean to say the output of ```ONNXLIRTorch(torch_model)```?
Maybe use the same naming style as below examples-`opt_model`?